### ISSUE TITLE
Scanner GUI Button Bug

### ISSUE THREAD

**Issue by trienow** [rwtema/extrautilities_old_issues#1286]
# What happens
![Screenshot of the Scanner GUI not behaving.](http://i.imgur.com/rpRG9SU.png)
After 'setting to current block' the buttons like to huddle in the top left corner.
_I don't bite._

# Versions
|    Software     |   Version    |
|-----------------|--------------|
| Minecraft       |       1.10.2 |
| Forge           | 12.18.3.2185 |
| ExtraUtilities2 |        1.2.0 |
| JEI             |   3.14.2.401 |

Thanks :)

**Comment by Crusaderkvw** [comment]
Just wanting to let you guys know that i'm having this very same issue.  

My modlist: http://modlister.com/user/kvwerkhoven

**Comment by Dragoon614** [comment]
Have you figured out any way to circumvent this issue? I'm also having it.

**Comment by Polyana-Fontes** [comment]
I'm also having this issue. This looks so easy to fix...

**Comment by trienow** [comment]
> Have you figured out any way to circumvent this issue? I'm also having it.

Yes I have actually. Some blocks do actually work. I _think_ when thy have multiple properties, such as a Storage Drawer. I placed the Storage Drawer and pressed 'Set to Current Block'.
![The GUI not bugging out](https://cloud.githubusercontent.com/assets/8478950/23272102/403437ea-f9fa-11e6-8910-72cd2a0bcd8e.png)

Next I placeed the other block (in my case a chest) and pressed 'Set to Current Block' again. It worked until I reopened the GUI.
![The GUI working on the chest](https://cloud.githubusercontent.com/assets/8478950/23272140/6f4edc2e-f9fa-11e6-9f86-e79a650a7692.png)

Regarding the Storage Drawer: It works fine, until you cross-out a property, then either some of the buttons huddle up in the corner or they overlap with others. Scrolling / reopening the GUI fixes this.
![The GUI not working on the Storage Drawer](https://cloud.githubusercontent.com/assets/8478950/23272198/abd3ce52-f9fa-11e6-8150-4627b605a790.png)

**Comment by jkerr** [comment]
Also an issue with the current version in "Beyond" mod pack - tried with ender lily and wheat crop, both have the GUI issue in the top screenshot.

**Comment by PapiDimmi** [comment]
I’m having exactly the same issue.
![extra utilities 2 glitch](https://cloud.githubusercontent.com/assets/25916106/24528117/756cdbb2-15a5-11e7-8cd8-53c7664b4c8b.png)

I’m playing [FTB Presents Direwolf20 1.10](https://www.feed-the-beast.com/projects/ftb-presents-direwolf20-1-10), with OptiFine added to it. Maybe OptiFine has something to do with it?

**Comment by ghost** [comment]
I’m having exactly the same issue.
![extra utilities 2 glitch](https://cloud.githubusercontent.com/assets/25916106/24528117/756cdbb2-15a5-11e7-8cd8-53c7664b4c8b.png)

I’m playing [FTB Presents Direwolf20 1.10](https://www.feed-the-beast.com/projects/ftb-presents-direwolf20-1-10), with OptiFine added to it. Maybe OptiFine has something to do with it?

@rwtema Please help us!

**Comment by TheOldOne822** [comment]
I just tested with just ExU and the gui is broken in all version from 1.2 onward, only work around I've found is to downgrade to ExU 1.1.3 but that removes many features.

**Comment by ghost** [comment]
The scanner is definitely not worth it to downgrade.

I _can_ use the scanner, but I cannot change the properties of the block.

I really hope that @rwtema fixes this. The scanner is practically unusable if you want to configure it.

**Comment by trienow** [comment]
@PapiDimmi I think we get the point by now. If you have anything else to say, please make it constructive, by for instance adding new information. Also tagging Tema in each of your posts, won't make anything get fixed faster.

**Comment by ghost** [comment]
@trienow That’s very non-constructive.

If you have anything to say that’s unrelated to this bug, please refrain from saying it. GitHub is not a forum board.

I certainly hope that @rwtema fixes this bug soon. Thanks, @rwtema. @rwtema.

**Comment by TheOldOne822** [comment]
I have now found a workaround but it may not be worth the effort. You can click the stacked buttons to toggle them all then use an NBT editor to delete the entry for the ones you want enabled.
